tests/conftest.py

```diff
@@ -500,11 +500,12 @@ def create_dagrun(self, **kwargs):
 
             dag = self.dag
             kwargs = {
-                "state": State.RUNNING,
                 "start_date": self.start_date,
                 "session": self.session,
                 **kwargs,
             }
+            if "state" not in kwargs:
+                kwargs["state"] = State.RUNNING
```

Comment:
This should not be needed, the dict unpacking syntax already guarantees `"state"` in `**kwargs` takes precedence over the one above it.